typed/typed.go

```diff
@@ -206,7 +202,7 @@ func (tv TypedValue) NormalizeUnionsApply(new *TypedValue) (*TypedValue, error)
 }
 
 func (tv TypedValue) Empty() *TypedValue {
-	tv.value = value.Value{Null: true}
+	tv.value = nil
```

Comment:
Shouldn't this use a value that is explicitly a Null?

Reply:
Clearly something I missed